pkg/kubelet/prober/worker.go

```diff
@@ -62,6 +63,11 @@ type worker struct {
 	resultRun int
 }
 
+const (
+	// jitter factor for worker.doProbe sync loop
+	workerProbeJitterFactor = 1.0
```

Comment:
The probers and the pod workers (for syncing) are the two main sources of goroutines in kubelet. The rest is in cadvisor.